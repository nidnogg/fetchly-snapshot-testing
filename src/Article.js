const Article = ({content}) => {
    return (
        <>
            <p>{content}</p>
        </>
    )
}
export default Article