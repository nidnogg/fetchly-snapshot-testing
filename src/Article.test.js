import renderer from 'react-test-renderer'
import Article from './Article.js'

it ('successful render', () => {
    const tree = renderer
        .create(<Article content="Lorem Ipsum Dolor Amet Version 2" />)
        .toJSON()
    expect(tree).toMatchSnapshot()
})