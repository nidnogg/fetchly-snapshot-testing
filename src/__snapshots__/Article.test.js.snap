// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`successful render 1`] = `
<p>
  Lorem Ipsum Dolor Amet
</p>
`;
